pkg/cloudprovider/providers/vsphere/nodemanager.go

```diff
@@ -317,3 +329,23 @@ func (nm *NodeManager) GetVSphereInstance(nodeName k8stypes.NodeName) (VSphereIn
 	}
 	return *vsphereInstance, nil
 }
+
+// refreshNodeInfo refreshes vSphere connection, VirtualMachine and Datacenter for NodeInfo instance.
```

Comment:
The comment should start with function name, i.e. refreshNodeInfo => renewNodeInfo

Reply:
Done